locations/spiders/cava.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+from locations.hours import OpeningHours
+
+STATES = {
+    "AK": "Alaska",
+    "AL": "Alabama",
+    "AR": "Arkansas",
+    "AS": "American Samoa",
+    "AZ": "Arizona",
+    "CA": "California",
+    "CO": "Colorado",
+    "CT": "Connecticut",
+    "DC": "District of Columbia",
+    "DE": "Delaware",
+    "FL": "Florida",
+    "GA": "Georgia",
+    "GU": "Guam",
+    "HI": "Hawaii",
+    "IA": "Iowa",
+    "ID": "Idaho",
+    "IL": "Illinois",
+    "IN": "Indiana",
+    "KS": "Kansas",
+    "KY": "Kentucky",
+    "LA": "Louisiana",
+    "MA": "Massachusetts",
+    "MD": "Maryland",
+    "ME": "Maine",
+    "MI": "Michigan",
+    "MN": "Minnesota",
+    "MO": "Missouri",
+    "MS": "Mississippi",
+    "MT": "Montana",
+    "NC": "North Carolina",
+    "ND": "North Dakota",
+    "NE": "Nebraska",
+    "NH": "New Hampshire",
+    "NJ": "New Jersey",
+    "NM": "New Mexico",
+    "NV": "Nevada",
+    "NY": "New York",
+    "OH": "Ohio",
+    "OK": "Oklahoma",
+    "OR": "Oregon",
+    "PA": "Pennsylvania",
+    "PR": "Puerto Rico",
+    "RI": "Rhode Island",
+    "SC": "South Carolina",
+    "SD": "South Dakota",
+    "TN": "Tennessee",
+    "TX": "Texas",
+    "UT": "Utah",
+    "VA": "Virginia",
+    "VI": "Virgin Islands",
+    "VT": "Vermont",
+    "WA": "Washington",
+    "WI": "Wisconsin",
+    "WV": "West Virginia",
+    "WY": "Wyoming",
+}
+
+DAY_MAPPING = [
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday",
+]
+
+
+class CavaSpider(scrapy.Spider):
+    name = "cava"
+    allowed_domains = ["www.cava.com"]
+    item_attributes = {"brand": "Cava", "brand_wikidata": "Q85751038"}
+    start_urls = ("https://cava.com/locations/",)
+
+    def parse(self, response):
+        state_selectors = response.xpath(
+            './/div[@class="menu-panel-wrapper"]/div[@class="menu-content"]'
+        )
+        for state_selector in state_selectors:
+            yield from self.parse_state(state_selector)
+
+    def parse_state(self, state):
+        location_selectors = state.xpath('.//div[@class="vcard"]')
+        for location_selector in location_selectors:
+            yield from self.parse_location(location_selector)
+
+    def parse_location(self, location):
+        nonBreakingSpace = "\xa0"
+        name = location.xpath(".//h3/text()").extract_first()
+        street_address = (
+            location.xpath('.//div[@class="street-address"]/text()')
```

Comment:
`street_address` should only be the `street-address` data and not include the locality.

Reply:
@iandees, the street address currently does not include any specific locality details.